book/website/afterword/glossary.md

```diff
@@ -405,8 +405,8 @@ Open Scholarship
 Open Project
   *Same as Open Science or Open Research Projects*. A project in which a significant amount of collaboration between the core or leadership team and the wider community takes place in the form of online interactions. Community interactions should maintain transparency and openness of the project to facilitate the growth of your community.
 
-Open Source Software
-  Documenting research code and routines, and making them freely accessible and available.
+Open Source
+  Open source software is software whose source code can be freely used, modified, and shared (in both modified and unmodified form). Today the concept of "open source" is often extended beyond software, to represent a philosophy of collaboration in which source files of digital resources are made available online to be used, modified and shared. (Ex: open source resource, open source educational resource, open source image, etc.)
 
```

Comment:
> I think we should have an OSS definition, in addition to a broader definition for open source. That said, I think we can improve the existing definition.

I also thought it deserves a refresh, so I did it at the same time through this merge, but a separate definition could also be a possibility. A counter-proposal taking on board some of the criticisms. I also invite you to read the article « [Defining Open Source](https://the.webm.ink/defining-open-source) by an OSI member working on standards on their approach to construct a definition of open source (in a dictionary/regular format).

**A definition basis:**
> Software distributed with its source code which provides the right to be {freely?} used, modified and shared, usually through open licenses. The [Open Source Definition](https://opensource.org/osd) outlines a set of licensing criteria that serve as the primary reference for defining open source. [...]

It reduces the emphasis on licenses as they may not be required for this kind of practice where it's the global copyright framework that influence provided freedom. Your proposal is an open source definition based on the OSD, it's reworded so as not to impose it as an immutable standard to deal with dissensus and be more inclusive of possible interpretations. It depends on how things are oriented regarding the OSD, freely qualifier is also sign of an OSI flavour as these kinds of things may come with some restrictions. This makes the source code aspect explicit, primary element before even considering the copyright dimension.

I thought it would be a good idea to merge the definitions in order to bring these meanings together, with the ambiguity that this can give rise to, starting with this standard usage of open source and moving towards this more exotic meaning. On the one hand, they are not separate concepts, since open source resources cover software. But then it's a design choice, the fusion looks cooler to me but I'm fine with both.

In any case, defining open source in this way is a tricky business, which is why I was so evasive in my initial proposals! Not sure if you expect to create it from scratch, or adapt/reuse an existing definition.

The one of the article by the OSI member:
> Open source software is software released under a license that — by broad community consensus — grants all rights necessary to use, adapt, share and monetise the software in any way and for any purpose subject only to conditions that can be reasonably satisfied without negotiation with the licensors.